components/Header/index.js

```diff
@@ -9,4 +9,36 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+
+
+function Header(item) {
+    //1. create elements - document.createElement()
+    const header = document.createElement('div');
+    const date = document.createElement('span');
+    const h1 = document.createElement('h1');
+    const temp = document.createElement('span');
+          
+    //2. .textContent
+    date.textContent = `SMARCH 28, 2019`;
+    h1.textContent = `Lambda Times`;
+    temp.textContent = `98°`;
+
+    //3. .classList.add()
+    header.classList.add('header');
+    date.classList.add('date');
+    temp.classList.add('temp');
+
+    //4. .appendChildren()
+    header.appendChild(date);
+    header.appendChild(h1);
+    header.appendChild(temp);
+
+    const headerContainer = document.querySelector('.header-container');
```

Comment:
remember you are inside your element creator function. do you want to be selecting INSIDE of this function?